crates/egui/src/data/input.rs

```diff
@@ -176,8 +176,11 @@ pub enum ViewportEvent {
 #[derive(Clone, Debug, Default, PartialEq)]
```

Comment:
If we should store the `ViewportId` in this `struct`, we should force users to initialize it to the correct thing when they create it via a `ViewportInfo::new`, otherwise it will be wrong.

```suggestion
#[derive(Clone, Debug, PartialEq)]
```